openml-h2o/src/main/java/com/feedzai/openml/h2o/server/H2OApp.java

```diff
@@ -326,7 +327,12 @@ private M waitForModelTrained(final Job<M> modelJob) {
      * @implNote This method needs to be synchronized. While we can't really pinpoint the exact reason, the experience
      * with tests training multiple models concurrently shows that this method is not thread safe.
      */
-    private synchronized Frame parseDataSetFile(final Path dataSetFile, final DatasetSchema schema) {
+    private synchronized Frame parseDataSetFile(final Path dataSetFile,
+                                                final DatasetSchema schema) throws ModelTrainingException{
```

Comment:
can you add a @throws to this method javadoc?

Reply:
Done. Thanks